docs/dev/handbook/cluster_creation.md

```diff
@@ -0,0 +1,187 @@
+Creating clusters with cluster-class
+========================================
+
+This developer handbook describe how to create a cluster using a clusterclass based CRD. 
+
+
+## Set up and host a package repository for management cluster
+First, make sure you have the push access to an OCI registry. Export its URL as `OCI_REGISTRY` environment variable.
+
+### Set up an OCI Registry with GCP
```

Comment:
This should work on any registry, for example it is working on OCIRegistry as well, so may as well remove the GCP specific wording.

Reply:
yes, any registry should work in theory (we should mention that). Here we provide GCP registry as an example.